### PR TITLE
fix(core): fix bullet demo issue

### PR DESCRIPTION
**Pull request by sophiiae**

### CODE REVIEW

**Review by natashadecoste — APPROVED** [review]
🎉

File: packages/core/src/interfaces/enums.ts [hunk 1]
```diff
@@ -214,7 +214,7 @@ export enum AxisTitleOrientations {
 /**
  * enum of legend item type
  */
-export enum LegendItemType {
+export enum LegendItemTypes {
```
Comment (cal-smith) [inline packages/core/src/interfaces/enums.ts:217]: Just want to make sure the `LegendItemType` -> `LegendItemTypes` change is something that we want, and isn't something that's going to trip up consumers? I might be missing something, but it just seems like an aesthetic change?

File: packages/core/src/tools.ts [hunk 2]
```diff
@@ -295,25 +297,28 @@ export namespace Tools {
 		);
 
 		// Retain default legend additional items
-		if (defaultAdditionalItems && userProvidedAdditionalItems) {
+		if (defaultAdditionalItems && userProvidedAdditionalItems) {
```
Comment (cal-smith) [inline packages/core/src/tools.ts:300]: Weird extra space here, not sure why prettier didn't catch that ...
```suggestion
		if (defaultAdditionalItems && userProvidedAdditionalItems) {
```

**Review by cal-smith — APPROVED** [review]
Looks good overall, just a couple questions

**Pull request merged**